### PR TITLE
Template

### PR DESCRIPTION
**Pull request by Certifieddonnie**
refactor: changed the folder name
fix: compiler fixes and code bug fixes

### CODE REVIEW

File: certifieddonnie/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Ignore files
+
+*.o
```
Comment (MartyOfMCA) [inline certifieddonnie/.gitignore:3]: Kindly add `.exe` files as well

**Review by MartyOfMCA — CHANGES_REQUESTED** [review]

File: certifieddonnie/.gitignore (hunk 1 above)
Comment (Certifieddonnie) [reply certifieddonnie/.gitignore:3]: Ohk Noted💯

Comment (MartyOfMCA) [reply certifieddonnie/.gitignore:5]: Great work 👍

File: certifieddonnie/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+- Error Handlers: The Program has ready made Error handling functions that can help when creating more functions. The Error handlers can be modified to your taste. Feel free to edit it and make them yours.
+- Betty Documentation Styles: The Program follows Betty Documentation styling. All functions are well documented!
+- Ignored Files: To avoid the folder to be clumsy. Object files are being ignored. Likewise other confidential files that are meant to be private to only you.
+
+## **GETTING STARTED**
+
+Irrespective of your System O.S, Follow these steps:
+
+1. Clone the repository -> [Demo-Project-Repository](https://github.com/ALX-SE-Algorithmia/Demo-Project)
+2. Change Directory to `Demo-Project/Certifieddonnie-C`
```
[the remark this answers is not in the document]
Comment (MartyOfMCA) [reply certifieddonnie/README.md]: Looks perfect now

**Review by MartyOfMCA — APPROVED** [review]
Great work @Certifieddonnie

**Comment by Certifieddonnie** [comment]
Thanks for the review Boss🎉 @MartyOfMCA

**Pull request merged**